src/form/bf_mx.jl

```diff
@@ -362,14 +167,14 @@ end
 
 "Defines voltage drop over a branch, linking from and to side voltage"
 function constraint_tp_model_voltage_magnitude_difference(pm::_PMs.GenericPowerModel{T}, n::Int, i, f_bus, t_bus, f_idx, t_idx, r, x, g_sh_fr, b_sh_fr, tm) where T <: AbstractUBFForm
-    w_fr_re = _PMs.var(pm, n, :W_re)[f_bus]
-    w_fr_im = _PMs.var(pm, n, :W_im)[f_bus]
+    w_fr_re = _PMs.var(pm, n, :Wr)[f_bus]
```

Comment:
i would probably name these variables like `Wr_fr`, `Wi_fr`, etc. for consistency. I did not mark all occurrences, please check